conans/test/integration/toolchains/gnu/test_autotools_toolchain.py

```diff
@@ -0,0 +1,56 @@
+import os
+import textwrap
+
+import pytest
+
+from conans.model.ref import ConanFileReference
+from conans.test.assets.autotools import gen_makefile_am, gen_configure_ac
+from conans.test.assets.sources import gen_function_cpp
+from conans.test.utils.tools import TurboTestClient
+
+
+@pytest.mark.tool_autotools()
+def test_install_output_directories():
```

Comment:
This test should probably go to ``functional`` as it is a full compilation test.